caproto/curio/server.py

```diff
@@ -312,42 +312,98 @@ def __init__(self, host, port, pvdb, *, log_level='ERROR'):
                 commands = self.broadcaster.recv(bytes_received, address)
                 await self.command_bundle_queue.put((address, commands))
 
-    async def broadcaster_queue_loop(self):
+    def __getitem__(self, pvname):
```

Comment:
Defining `__getitem__` also provides an implicit `__iter__`, maybe use a normal method there?

Reply:
or support more of the dict interface.